doc/src/access_solutions.rst

```diff
@@ -0,0 +1,48 @@
+.. _Access Solutions:
+
+Programmatic Access to Solutions
+================================
+
+The mechanisms for outputting and accessing solutions depends on how the solutions
+were obtained. We will describe some of the possibilities for programmatic access
+to the solutions here.
+
+EF
+--
+
+If you are not using a ``WheelSpinner`` object, but rather creating an
+``EF`` object directly,
+you can use the function ``mpisppy.sputils.ef_scenarios`` to loop over
+the scenario models after sololution. The function takes one arugment,
+which is the EF object.
+
+For example, suppose you have an extensive form (EF) object, `ef`, that
+is of a type derived from SPBase (which is almost surely is), then you
+can access the ``Pyomo`` model for every scenario using something like:
+
+::
+   
+   for sname, smodel in sputils.ef_scenarios(ef):
+
+One can access the variables in the Pyomo model `smodel` inside the loop.
```

Comment:
Isn't this the same as `ef.scenarios.items()`? I wonder if we should just tell users about that.

There's also the fact that since it's derived from `SPBase`, we could point the user to `SPBase.write_tree_solution`, which is the same format as for `WheelSpinner`.